theme/templates/features.html

```diff
@@ -62,11 +63,11 @@ <h2 class="heading">{% trans %}Easily Customizable{% endtrans %}</h2>
         </div>
         <div class="col-xs-12 col-sm-6 col-md-6">
           <h2 class="heading">{% trans %}Fast{% endtrans %}</h2>
-          <p>{% trans %}With Tryton acting as the core of the system, all your tasks can be managed with simple clicks, getting results in the blink of an eye!{% endtrans %}
+          <p>{% trans %}With Tryton acting at the heart of the system, all tasks can be managed efficiently, getting desired results rapidly!{% endtrans %}
           </p>
-          <p>{% trans %}The Tryton kernel is ELEVEN TIMES FASTER, which means it can handle huge applications featuring colossal data sets at lightning fast speed. You spend more time in getting actual work done, rather than wasting your time on slow systems.{% endtrans %}
+          <p>{% trans %}The Tryton kernel is ELEVEN TIMES FASTER than its counterparts, which means it can handle huge applications featuring colossal data sets at lightning fast speeds. Thus, you can spend more time in getting actual work done, rather than wasting your time on slow systems.{% endtrans %}
```

Comment:
The Tryton kernel is FASTER than its counterparts, which means it can handle huge applications featuring colossal data sets at lightning fast speed.